docs/source/documentation/4-docsearch-FAQ/1-customize-configuration-file.html.md.erb

```diff
@@ -0,0 +1,71 @@
+---
+title: Customize my configuration file
+---
+
+## What content do we recommend indexing?
+
+### Code blocks
+
+We **do not recommend** indexing **code tags** since code blocks are most likely
+very similar and redundant from one page to another.
+You can exclude code blocks by adding their matching selectors to `selectors_exclude`.
+
+A good practice would be to emphasize the meaningful underlying part of it
```

Comment:
I don't understand what is the "meaningful underlying part" of a code block? Maybe we can provide an example of a code block that has meaningful content?

Reply:
The included part that you want to highlight:
 Example: the params of a code tag
<code>
blabla(<toIndex>something</toIndex>)
...
</code>